packages/web/components/buttons/claim-all-rewards.tsx

```diff
@@ -0,0 +1,22 @@
+import { FunctionComponent } from "react";
+
+import { Button } from "~/components/buttons/button";
+import { ToggleProps } from "~/components/control";
+
+export const ClaimAllRewardsButton: FunctionComponent<ToggleProps> = ({
+  onToggle,
+}) => {
+  // TODO: Use translations
+  //const {t} = useTranslation();
+  return (
+    <div className="max-w-xs">
+      <Button
+        mode="secondary"
+        className="mt-8 gap-3 whitespace-nowrap !px-6"
+        onClick={() => onToggle(false)}
+      >
+        Claim All Rewards
```

Comment:
FYI it's possible to add a key to en.json (somewhere that makes sense) and run `yarn machine-translate && yarn lint:fix` in the root of the repo to generate translations